src/main/java/org/jenkinsci/test/acceptance/po/JenkinsDatabaseSecurityRealm.java

```diff
@@ -81,7 +83,7 @@ public Signup email(String mail) {
 
         public User signup(String name) {
             control(by.input("username")).set(name);
-            control(by.name("Submit")).click();
+            control(by.name("Submit")).clickAndWaitToBecomeStale(Duration.ofSeconds(120));
```

Comment:
2 minutes? Seems too much, I think 1 or 2 seconds should be more than enough.

Reply:
lol sorry, I didn't pushed the last commit :facepalm: 

now it should be using the default timeout of `clickAndWaitToBecomeStale`, which is 30s